tests/test_project_time.py

```diff
@@ -86,6 +87,21 @@ def test_convert_to_project_time(registry_mgr):
         raw_data=tempo_data_with_tz,
         converted_data=tempo_data_mapped_time,
     )
+    compare_time_conversion(
+        resstock_time_dim, project_time_dim, wrap_time=False, expect_error=True
+    )
+    compare_time_conversion(
+        comstock_time_dim, project_time_dim, wrap_time=False, expect_error=True
+    )
+    compare_time_conversion(
+        resstock_time_dim, project_time_dim, wrap_time=True, expect_error=False
+    )
+    compare_time_conversion(
+        comstock_time_dim, project_time_dim, wrap_time=True, expect_error=False
+    )
```

Comment:
Are the resstock and comstock time dimensions identical? If so, should we just test one of them?